test/e2e_node/pod_resize_test.go

```diff
@@ -360,6 +366,60 @@ func verifyPodContainersCgroupValues(ctx context.Context, f *framework.Framework
 	return nil
 }
 
+// Credits github.com/prometheus/procfs/proc_status.go
+func calcCpusAllowedList(cpuString string) []uint64 {
```

Comment:
I refactored pod resize e2e tests in #128143. Is it possible to update the test file?

Reply:
Thanks @AnishShah , nice job! Absolutely working on it!

Reply:
Hi, @AnishShah just to let you know that there was a reason why i had pod_resize tests under e2e_node, it was to reuse CPU Manager functions necessary to perform pod_resize tests along Static Policy, those  exist under e2e_node. Now with the refactoring i cannot use them because i  get import cycle not allowed errors, thus i have to move those functions to the new common .  This will take time, i want first to prioritize the introduction of feature gate first.  Will ping when it is done.

PS: I don't understand why you have added this comment only on line 369 ? My commit does much more when it comes to tests, introduces ability to modify kubelet CPU manager config, hundreds of tests for all combinations etc.

Reply:
@esotsal , Apologies. My comment was for the whole file. After my PR, the file was deleted and some test utilities were moved to [`test/e2e/framework/pods/resize.go`](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/pod/resize.go) so that it can be used in both e2e_node and cluster tests. I agree with you that cpu manager tests can stay in `e2e_node`. I meant rebasing your test file and reusing test utilities from [`test/e2e/framework/pods/resize.go`](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/pod/resize.go). Sorry

Reply:
Np

Reply:
@AnishShah updated the tests, and now using your refactored approach. Seems it works great, had two successful runs ( [1](https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/directory/pull-kubernetes-node-kubelet-serial-podresize/1848756510969040896), [2](https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/directory/pull-kubernetes-node-kubelet-serial-podresize/1848721841451962368)) :-)

Reply:
Setting back to unresolved, until i push a new patch. Cpu manager test must stay to  e2e_node tests, tried to refactor but hit a dead end trying to write the kubelet configs in cluster. Since Beta is both urgent ( less than a week for code freeze ) and important will push the refactoring after v1.32

Reply:
@AnishShah PTAL ,in order to test #128287  tests, refactoring was needed before v1.32 . Looking forward for your review in #128287 .